### PR TITLE
Pass through aria-* attributes

### PR DESCRIPTION
**Pull request by mbrodala**

### CODE REVIEW

**Review by jonaseberle — APPROVED** [review]

File: src/Core/ViewHelper/AbstractTagBasedViewHelper.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function initialize()
         }
 
         foreach ($this->additionalArguments as $argumentName => $argumentValue) {
-            if (strpos($argumentName, 'data-') === 0) {
+            if (strpos($argumentName, 'data-') === 0 || strpos($argumentName, 'data-') === 0) {
```
Comment (o-ba) [inline src/Core/ViewHelper/AbstractTagBasedViewHelper.php:113]: This should be `strpos($argumentName, 'aria-')`

File: tests/Unit/Core/ViewHelper/AbstractTagBasedViewHelperTest.php [hunk 2]
```diff
@@ -153,6 +153,26 @@ public function testValidateAdditionalArgumentsThrowsExceptionIfContainingNonDat
...
+            [],
+            '',
+            false
+        );
+        $viewHelper->setRenderingContext(new RenderingContextFixture());
+        $tagBuilder = $this->getMock(TagBuilder::class, ['addAttribute']);
+        $tagBuilder->expects($this->at(0))->method('addAttribute')->with('aria-labelledby', 'foo');
+        $tagBuilder->expects($this->at(1))->method('addAttribute')->with('aria-live', 'polite');
+        $viewHelper->setTagBuilder($tagBuilder);
+        $viewHelper->handleAdditionalArguments(['aria-labelledby' => 'foo', 'aria-live' => 'polite']);
```
Comment (o-ba) [inline tests/Unit/Core/ViewHelper/AbstractTagBasedViewHelperTest.php:173]: `$viewHelper->initializeArgumentsAndRender();` must be added here to fix the test.

File: src/Core/ViewHelper/AbstractTagBasedViewHelper.php (hunk 1 above)
Comment (mbrodala) [reply src/Core/ViewHelper/AbstractTagBasedViewHelper.php:113]: Yikes, a total rebase failure. :-(

**Review by o-ba — APPROVED** [review]

**Comment by mbrodala** [comment]
@NamelessCoder do you approve, too? Otherwise I cannot merge.

**Comment by NamelessCoder** [comment]
This is redundant after merging https://github.com/TYPO3/Fluid/pull/536, right?

**Comment by mbrodala** [comment]
Actually my idea was to merge this first and then the other one for the argument. But now it's done already, so we can close this.

**Pull request closed without merging**